### PR TITLE
Minor Improvements

### PR DESCRIPTION
**Pull request by zachlatta**

### CODE REVIEW

File: workshops/dodge/README.md [hunk 1]
```diff
@@ -19,34 +19,65 @@ p5play is made by a really cool game company called [Molleindustria](http://www.
...
-<script src="https://cdn.rawgit.com/molleindustria/p5.play/master/lib/p5.play.js"></script>
-<script src="game.js"></script>
-```
+Our project's code is going to be divided into two files: `index.html` (the HTML code) and `game.js` (the JavaScript code). The HTML file will tell the browser about the game's existence and how to display it on the page. The JavaScript file will tell the browser how to actually run the game.
 
-The first line grabs the p5js library from where it's hosted on Cloudflare. The second grabs the p5play library that we'll need to build our game. And the third tells our HTML file about the external JS file where we'll write our game.
+1. Create a new directory in your Cloud9 workspace called `dodge`
+2. Create a file and copy and paste the following into it. Save it as `index.html` inside of the `dodge` directory.
 
-Create a new file named `index.html` in your directory, and throw those `<script>` tags in the `<body>`. Spin up the live preview and prepare to be blown away ... by nothing.
```
Comment (jkwok91) [inline workshops/dodge/README.md:32]: YO I HAD A NICE NARRATIVE GOIN ON HERE :( Why did you remove my rhetorical flair/humor :sob: :sob: :sob:

File: workshops/dodge/README.md [hunk 2]
```diff
@@ -19,34 +19,65 @@ p5play is made by a really cool game company called [Molleindustria](http://www.
...
+    </body>
+    </html>
+    ```
 
-That's it for the HTML page! Next we'll create that external JS file we just referred to.
+     The two `script` tags inside of `body` tell the browser to load p5.js and p5.play _(improve this introduction)_
+ 
+3. Create another new file and then save it as `game.js` inside of the `dodge` directory. Leave it empty for now.
+ 
+Great, so now we have `index.html` and `game.js` created. `index.html` is loading p5.js and p5.play, but it doesn't know about `game.js` yet. Let's change that.
```
Comment (jkwok91) [inline workshops/dodge/README.md:46]: :+1:

File: workshops/dodge/README.md [hunk 3]
```diff
@@ -19,34 +19,65 @@ p5play is made by a really cool game company called [Molleindustria](http://www.
...
+</head>
+<body>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/p5.js/0.4.12/p5.js"></script>
+  <script src="https://cdn.rawgit.com/molleindustria/p5.play/master/lib/p5.play.js"></script>
+  <script src="game.js"></script>
+</body>
+</html>
+```
+
+While you have `index.html` open, click `Preview > Live Preview File (index.html)` to see what the game looks like so far. Right now it's just a blank page, but we'll be changing that soon enough :wink:.
```
Comment (jkwok91) [inline workshops/dodge/README.md:64]: Ugh, so positive.

File: workshops/dodge/README.md [hunk 4]
```diff
@@ -64,50 +95,49 @@ Now we'll add something for the `draw()` function to do as well. Let's start by
 
 ```
 function draw() {
-  background(0,0,100);
+  background(0, 0, 100);
 }
 ```
 
-`background()` is a function that sets the appearance of the background. Here we're using three arguments to define the RGB values of a color, which will be the background color of our canvas. You can use background in some more ways, though, which are detailed in documentation [here](https://p5js.org/reference/#/p5/background).
+`background()` is a function that sets the appearance of the background. Here we're using three arguments to define the RGB values of a color, which will be the background color of our canvas. You can use `background` in some more ways, though, which are detailed in documentation [here](https://p5.js.org/reference/#/p5/background).
```
Comment (jkwok91) [inline workshops/dodge/README.md]: NO why did you not `c` your replace command

File: workshops/dodge/README.md [hunk 5]
```diff
@@ -64,50 +95,49 @@ Now we'll add something for the `draw()` function to do as well. Let's start by
...
 function setup() {
-  createCanvas(500,500);
-  player = createSprite(width/2,height,50,50);
+  createCanvas(500, 500);
+  player = createSprite(width/2, height, 50, 50);
 }
 ```
 
-[`createSprite()`](http://p5play.molleindustria.org/docs/classes/p5.play.html#method-createSprite) makes a placeholder for an image (meaning you can customize with your own!), and takes four arguments (horizontal and vertical coordinates, and width and height of sprite, respectively).
+[`createSprite()`](http://p5.play.molleindustria.org/docs/classes/p5.play.html#method-createSprite) makes a placeholder for an image (meaning you can customize with your own!), and takes four arguments (horizontal and vertical coordinates, and width and height of sprite, respectively).
```
Comment (jkwok91) [inline workshops/dodge/README.md]: NO

File: workshops/dodge/README.md [hunk 6]
```diff
@@ -286,9 +315,9 @@ Et voilà! Our enemy now shows up randomly across the top of the canvas! And dro
 
 Well, if you've failed to dodge the enemy at least once, you'll realize that the player is not forced to frantically dodge after all... the enemy doesn't attack it at all.
 
-Let's make it a real threat by recognizing when the enemy and player collide. P5play provides the `overlap()` method on our sprites for doing just that.
+Let's make it a real threat by recognizing when the enemy and player collide. P5.Play provides the `overlap()` method on our sprites for doing just that.
```
Comment (jkwok91) [inline workshops/dodge/README.md]: weird caps

File: workshops/dodge/README.md [hunk 7]
```diff
@@ -298,16 +327,16 @@ This is our lose condition. Now we can punish the user for losing in this `gameO
...
+Even though it may feel like we've been writing all of our code in these two special p5.js functions to make stuff in the canvas, we musn't forget that we are still just writing JavaScript. So let's define our own function, `gameOver()`, the way we would any other JS function. We'll place it underneath the `draw()` function.
 
-```
+```js
 function gameOver() {
 }
 ```
 
-And let's display a Game Over screen.
+And let's display a game over screen.
```
Comment (jkwok91) [inline workshops/dodge/README.md:337]: Veto. Either "Game Over" or game-over

File: workshops/dodge/README.md (hunk 4 above)
Comment (zachlatta) [inline workshops/dodge/README.md]: Oh no, good catch. Let me fix that.

On Wed, Apr 6, 2016 at 9:18 AM, JESSICA notifications@github.com wrote:

> In workshops/dodge/README.md
> https://github.com/jkwok91/hackclub/pull/2#discussion_r58735764:
> 
> >  }
> > 
> > ```
> > 
> > -`background()` is a function that sets the appearance of the background. Here we're using three arguments to define the RGB values of a color, which will be the background color of our canvas. You can use background in some more ways, though, which are detailed in documentation [here](https://p5js.org/reference/#/p5/background).
> > +`background()` is a function that sets the appearance of the background. Here we're using three arguments to define the RGB values of a color, which will be the background color of our canvas. You can use `background` in some more ways, though, which are detailed in documentation [here](https://p5.js.org/reference/#/p5/background).
> > ```
> 
> NO why did you not c your replace command
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> https://github.com/jkwok91/hackclub/pull/2/files/f24ad8246cdf7e3e53e56b84653a167ecb9fe63a#r58735764

## 

Zach Latta
https://zachlatta.com http://zachlatta.com

File: workshops/dodge/README.md (hunk 6 above)
Comment (zachlatta) [inline workshops/dodge/README.md]: Yeah, gotta fix that.

On Wed, Apr 6, 2016 at 10:38 AM, JESSICA notifications@github.com wrote:

> In workshops/dodge/README.md
> https://github.com/jkwok91/hackclub/pull/2#discussion_r58748739:
> 
> > @@ -286,9 +315,9 @@ Et voilà! Our enemy now shows up randomly across the top of the canvas! And dro
> > 
> >  Well, if you've failed to dodge the enemy at least once, you'll realize that the player is not forced to frantically dodge after all... the enemy doesn't attack it at all.
> > 
> > -Let's make it a real threat by recognizing when the enemy and player collide. P5play provides the `overlap()` method on our sprites for doing just that.
> > +Let's make it a real threat by recognizing when the enemy and player collide. P5.Play provides the `overlap()` method on our sprites for doing just that.
> 
> weird caps
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> https://github.com/jkwok91/hackclub/pull/2/files/f24ad8246cdf7e3e53e56b84653a167ecb9fe63a#r58748739

## 

Zach Latta
https://zachlatta.com http://zachlatta.com

File: workshops/dodge/README.md (hunk 7 above)
Comment (zachlatta) [inline workshops/dodge/README.md:337]: Well, "Game Over" is not a proper noun. So shouldn't be capitalized. Not
sure why we'd style it as game-over either.

On Wed, Apr 6, 2016 at 10:39 AM, JESSICA notifications@github.com wrote:

> In workshops/dodge/README.md
> https://github.com/jkwok91/hackclub/pull/2#discussion_r58748832:
> 
> >  function gameOver() {
> >  }
> > 
> > ```
> > 
> > -And let's display a Game Over screen.
> > +And let's display a game over screen.
> > ```
> 
> Veto. Either "Game Over" or game-over
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> https://github.com/jkwok91/hackclub/pull/2/files/f24ad8246cdf7e3e53e56b84653a167ecb9fe63a#r58748832

## 

Zach Latta
https://zachlatta.com http://zachlatta.com

File: workshops/dodge/README.md (hunk 1 above)
Comment (zachlatta) [inline workshops/dodge/README.md:32]: It wasn't clear how to set up the project in Cloud9.

File: workshops/dodge/README.md (hunk 7 above)
Comment (jkwok91) [inline workshops/dodge/README.md:337]: well then "game over"
but regardless, at least make the change throughout the document

Comment (jkwok91) [inline workshops/dodge/README.md:337]: Nevermind, I see you've done that
